apps/meteor/app/authorization/server/constant/permissions.ts

```diff
@@ -83,7 +83,7 @@ export const permissions = [
 	{ _id: 'preview-c-room', roles: ['admin', 'user', 'anonymous'] },
 	{ _id: 'view-outside-room', roles: ['admin', 'owner', 'moderator', 'user'] },
 	{ _id: 'view-broadcast-member-list', roles: ['admin', 'owner', 'moderator'] },
-	{ _id: 'call-management', roles: ['admin', 'owner', 'moderator'] },
+	{ _id: 'call-management', roles: ['admin', 'owner', 'moderator', 'user'] },
```

Comment:
should there not be a migration for this?

Reply:
There's already a migration calling upsertPermissions in there. If this PR gets left out of the release I'll include a new one.